extensions/wikia/CuratedContent/CuratedContentHelper.class.php

```diff
@@ -0,0 +1,163 @@
+							$item['type'] = self::STR_EMPTY_CATEGORY;
+						}
+					} else {
+						$item['type'] = null;
+					}
+					break;
+			}
+
+			if ( empty( $imageId ) ) {
+				$imageTitle = self::findFirstImageTitleFromArticle( $articleId );
+				if ( !empty( $imageTitle ) ) {
+					$imageId = $imageTitle->getArticleId();
+				}
+			}
+			$item['article_id'] = $articleId;
+			$item['image_id'] = $imageId;
+		}
+	}
+
+	public static function getIdFromCategoryName( $categoryName ) {
+		$category = Title::newFromText( $categoryName, NS_CATEGORY );
+
+		return ($category instanceof Title && $category->exists()) ? $category->getArticleID() : 0;
+	}
+
+	public static function getImageUrl( $id, $imageSize = 50 ) {
+		$thumbnail = (new ImageServing( [ $id ], $imageSize, $imageSize ))->getImages( 1 );
+
+		return !empty( $thumbnail ) ? $thumbnail[$id][0]['url'] : '';
+	}
+
+	private function getVideoInfo( $title ) {
+		$mediaService = new MediaQueryService();
+		$mediaInfo = $mediaService->getMediaData( $title );
+		if ( !empty( $mediaInfo ) ) {
+			if ( $mediaInfo['type'] === MediaQueryService::MEDIA_TYPE_VIDEO ) {
+				return [
+					'provider' => $mediaInfo['meta']['provider'],
+					'thumb_url' => $mediaInfo['thumbUrl'],
+					'videoId' => $mediaInfo['meta']['videoId']
+				];
+			}
+		}
+		return null;
+	}
+
+	public static function findImage( $imageId, $articleId = 0 ) {
+		$url = '';
+		$imageTitle = null;
+
+		if ( empty( $imageId ) ) {
+			$imageId = null;
+			$imageTitle = self::findFirstImageTitleFromArticle( $articleId );
+		} else {
+			$imageTitle = Title::newFromID( $imageId );
+		}
+		if ( $imageTitle instanceof Title && $imageTitle->exists() ) {
+			$url = self::getUrlFromImageTitle( $imageTitle );
+			$imageId = $imageTitle->getArticleId();
+		}
+
+		return [ $imageId, $url ];
+	}
+
+	public static function findFirstImageTitleFromArticle( $articleId ) {
+		$imageTitle = null;
+		if ( !empty( $articleId ) ) {
+			$imageServing = new ImageServing( [ $articleId ] );
+			$image = $imageServing->getImages( 1 );
+			if ( !empty( $image ) ) {
+				$imageTitleName = $image[$articleId][0]['name'];
+				if ( !empty( $imageTitleName ) ) {
+					$imageTitle = Title::newFromText( $imageTitleName, NS_FILE );
+				}
+			}
+		}
+		return ($imageTitle instanceof Title && $imageTitle->exists()) ? $imageTitle : null;
+	}
+
+	public static function getUrlFromImageTitle( $imageTitle ) {
+		$imageUrl = null;
+		if ( !empty( $imageTitle ) ) {
+			$imageFile = wfFindFile( $imageTitle );
+			if ( !empty( $imageFile ) ) {
+				$imageUrl = $imageFile->getUrl();
+			}
+		}
+		return $imageUrl;
+	}
+
+	public static function getTypeFromNamespaceId( $namespaceId ) {
+		switch ( $namespaceId ) {
+			case NS_MAIN:
+				return self::STR_ARTICLE;
+				break;
+			case NS_BLOG_ARTICLE:
+				return self::STR_BLOG;
+				break;
+			case NS_CATEGORY:
+				return self::STR_CATEGORY;
+				break;
+			case NS_FILE:
+				return self::STR_FILE;
+				break;
+			default:
+				return null;
```

Comment:
Defau,lt is null - validator checks if it's not null, so null equals wrong namespace.